apps/dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1,9 +1,17 @@
 // Load bootstrap/font-awesome sprockets first
-@import "bootstrap-sprockets";
+
+@import "variables";
+
+@import "../node_modules/bootstrap/scss/bootstrap";
```

Comment:
Since node_modules is still in the path, and we can deal with that for now (though need to definitely test performance due to this), it should work to do

    @import "bootstrap/scss/bootstrap";